XamlControlsGallery/App.xaml

```diff
@@ -40,6 +40,12 @@
                 </ResourceDictionary>
             </ResourceDictionary.ThemeDictionaries>
 
+            <LinearGradientBrush x:Key="HeroImageGradientBrush" StartPoint="0.5,0" EndPoint="0.5,1">
+                <GradientStop Offset="0" Color="{ThemeResource SystemAltMediumColor}"/>
```

Comment:
Nit: since the colors at offset 0 and offset 0.5 are the same, I don't think you need the one at offset 0. I tried it out and I _think_ it looks the same, but I might be missing something. Not important, just calling it out for FYI.